### PR TITLE
Fixes to ipset module

### PR DESCRIPTION
**Pull request by garethgreenaway**
Fixing issue when information returned from ipset isn't in the format we expect and it causes an exception.  Also fixing issues when specifying single IP address with a network mask, but ipset has stripped out the network mask so the matching fails. #26628

### CODE REVIEW

**Comment by basepi** [comment]
@garethgreenaway Looks like a lint failure or two: https://jenkins.saltstack.com/job/salt-pr-lint-n/8765/violations/file/salt/modules/ipset.py/

File: salt/modules/ipset.py [hunk 1]
```diff
@@ -392,6 +393,16 @@ def check(set=None, entry=None, family='ipv4'):
...
 
+    # If IPv4 address is a single host with a mask
+    # remote the /32, since ipset strips it.
+    if entry.endswith('/32'):
+        entry = re.sub('\/32', '', entry)
+
+    # If IPv6 address is a single host with a mask
+    # remote the /128, since ipset strips it.
+    if entry.endswith('/128'):
+        entry = re.sub('\/128', '', entry)
```
Comment (basepi) [inline salt/modules/ipset.py]: Needs to be `'\\/128'`, same with the `'\\/32'` above.

Comment (garethgreenaway) [inline salt/modules/ipset.py]: I'll probably redoing this, wasn't totally happy with the approach I took :)

Comment (basepi) [inline salt/modules/ipset.py]: Should I close this PR out then?

Comment (garethgreenaway) [inline salt/modules/ipset.py]: Yeah. I'll put another one together.

Comment (basepi) [inline salt/modules/ipset.py]: Great

**Pull request closed without merging**